src/siteconfig/models.py

```diff
@@ -199,6 +200,18 @@ class SiteConfig(models.Model):
             or \"Skill\" might be a more suitable name, depending on how you use the Tags feature."
     )
 
+    # Custom stylesheet and javascript
+    #
+    custom_stylesheet = RestrictedFileField(
+        verbose_name="CSS", null=True, blank=True,
+        content_types=['text/css', 'text/plain'], max_upload_size=512000,
+        help_text="Upload a custom CSS file to customize the look of your site.",
+    )
+    custom_javascript = RestrictedFileField(
+        verbose_name="JavaScript", null=True, blank=True,
```

Comment:
Remove the verbose and just use the default name, or verbose "Custom JavaScript" for the capitalization